stablehlo/reference/ProcessGrid.cpp

```diff
@@ -77,6 +77,8 @@ SmallVector<Tensor> RendezvousResult::getSortedTensors() const {
       llvm::map_range(result_, [](const auto &pair) { return pair.second; }));
 }
 
+size_t RendezvousResult::size() const { return result_.size(); }
```

Comment:
Is this used anywhere? If not lets remove and also remove the size method from thread safe map.

Reply:
(outdated, removed)